lib/groonga-query-log/command/run-regression-test.rb

```diff
@@ -56,6 +60,17 @@ def initialize
         @target_command_names = ServerVerifier::Options.new.target_command_names
 
         @read_timeout = Groonga::Client::Default::READ_TIMEOUT
+
+        @mail_notification = false
+        @mail_subject_on_succeeded = "Succeeded"
+        @mail_subject_on_failure = "Failed"
+        @mail_from = "groonga-query-log@example.com"
+        @mail_to = "noreply@localhost"
```

Comment:
It's not a good default value. We must require the value. `--mail-notification` may not needed.